object-manipulation-3/main.js

```diff
@@ -0,0 +1,117 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var players = [
+  { name: 'ObiWan', hand: [] },
+  { name: 'Anakin', hand: [] },
+  { name: 'Grievous', hand: [] },
+  { name: 'Droid', hand: [] }
+];
+
+var cards = [
+  { rank: 'Ace', suit: 'Diamonds' },
+  { rank: 'Ace', suit: 'Hearts' },
+  { rank: 'Ace', suit: 'Clubs' },
+  { rank: 'Ace', suit: 'Spades' },
+  { rank: 'King', suit: 'Diamonds' },
+  { rank: 'King', suit: 'Hearts' },
+  { rank: 'King', suit: 'Clubs' },
+  { rank: 'King', suit: 'Spades' },
+  { rank: 'Queen', suit: 'Diamonds' },
+  { rank: 'Queen', suit: 'Hearts' },
+  { rank: 'Queen', suit: 'Clubs' },
+  { rank: 'Queen', suit: 'Spades' },
+  { rank: 'Jack', suit: 'Diamonds' },
+  { rank: 'Jack', suit: 'Hearts' },
+  { rank: 'Jack', suit: 'Clubs' },
+  { rank: 'Jack', suit: 'Spades' },
+  { rank: '10', suit: 'Diamonds' },
+  { rank: '10', suit: 'Hearts' },
+  { rank: '10', suit: 'Clubs' },
+  { rank: '10', suit: 'Spades' },
+  { rank: '9', suit: 'Diamonds' },
+  { rank: '9', suit: 'Hearts' },
+  { rank: '9', suit: 'Clubs' },
```

Comment:
Try generating this deck with a nested loop. Create an array of ranks and an array of suits and then in the inner loop, push a card object into your cards array.